lib/bandit/http1/adapter.ex

```diff
@@ -58,6 +60,10 @@ defmodule Bandit.HTTP1.Adapter do
                connection: connection,
                keepalive: keepalive
            }}
+
```

Comment:
Related text in [RFC9112](https://www.rfc-editor.org/rfc/rfc9112)

> If a message is received with both a [Transfer-Encoding](https://www.rfc-editor.org/rfc/rfc9112.html#field.transfer-encoding) and a [Content-Length](https://www.rfc-editor.org/rfc/rfc9112.html#body.content-length) header field, the Transfer-Encoding overrides the Content-Length. Such a message might indicate an attempt to perform request smuggling ([Section 11.2](https://www.rfc-editor.org/rfc/rfc9112.html#request.smuggling)) or response splitting ([Section 11.1](https://www.rfc-editor.org/rfc/rfc9112.html#response.splitting)) and ought to be handled as an error. An intermediary that chooses to forward the message MUST first remove the received Content-Length field and process the Transfer-Encoding (as described below) prior to forwarding the message downstream.

Reply:
nice